source/tutorial/restart-sharded-cluster.txt

```diff
@@ -0,0 +1,37 @@
+=========================
+Restart a Sharded Cluster
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The tutorial is specific to MongoDB |version|. For earlier versions of
+MongoDB, refer to the corresponding version of the MongoDB Manual.
+
+This procedure demonstrates the shutdown and startup sequence for restarting
+a sharded cluster. Stopping or starting the components of a sharded cluster
+in a different order may cause communication errors between members. For
+example, :doc:`shard </core/sharded-cluster-shards>` servers may appear to hang
+if there are no :doc:`config servers </core/sharded-cluster-config-servers>`
+available.
+
+
+.. _shutdown-sharded-cluster:
+
+Stop a Sharded Cluster
+----------------------
```

Comment:
Suggest a pre-req or admonition that users perform this during a maintenance period during which applications stop all writes and reads to the cluster to prevent potential data loss or reading stale data.

Reply:
done